kanidmd/src/lib/ldap.rs

```diff
@@ -529,6 +530,8 @@ pub(crate) fn ldap_vattr_map(input: &str) -> &str {
         "sshpublickey" => "ssh_publickey",
         "cn" => "name",
         "uidnumber" => "gidnumber",
+        // TODO: temporary hack to make it work until the [kanidm::constants::ldap::LdapUacFlag] enum can be turned into a filter
```

Comment:
Why can't it be a filter right now?

Reply:
Because I don't know how to write them? 😅

Reply:
Isn't it the other way here? We need to synthesise the UACflags from other values?

Reply:
Yes, I don't know the right words for the thing, I just put in a hack to make it work for testing 😄  You mentioned elsewhere it'd need to be... oh, a plugin. 

Words are hard. Derp.